doc/classes/PointMesh.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="PointMesh" inherits="PrimitiveMesh" category="Core" version="3.2">
+	<brief_description>
+		Mesh with a single Point primitive.
+	</brief_description>
+	<description>
+		The PointMesh is made from a single point. Instead of relying on triangles, points 
+		are rendered as a single rectangle on the screen with a constant size. They are intended to be used with 
+		Particle systems, but can be used as a cheap way to render constant size billboarded sprites.
+
```

Comment:
There shouldn't be empty lines in the XML, each line translates to a paragraph. Similarly, no hard wrapping, keep paragraphs on a single line.